viewer.py

```diff
@@ -654,29 +767,50 @@ def __exit__(self, *args):
         self.signal.connect(self.slot)
 
 
-
 if __name__ == '__main__':
 
     app = pg.mkQApp()
 
     v = AtlasViewer()
     v.show()
 
-    path = os.path.dirname(__file__)
+    path = os.path.dirname(os.path.realpath(__file__))
     atlasFile = os.path.join(path, "ccf.ma")
     labelFile = os.path.join(path, "ccf_label.ma")
 
     if os.path.isfile(atlasFile):
         atlas = metaarray.MetaArray(file=atlasFile, readAllData=True)
     else:
-        atlas = readNRRDAtlas()
-        writeFile(atlas, atlasFile)
+        try:
+            atlas = readNRRDAtlas()
+            writeFile(atlas, atlasFile)
+        except:
+            try:
+                print "Unexpected error when creating ccf.ma file with " + atlasFile
+                print traceback.print_exc()
```

Comment:
We had a little bit of issues creating the .ma files while installing. So in order to make installation a little easier and less confusing, I added code to remove any .ma file if an error was encountered while creating ccf.ma or ccf_label.ma. But, in windows I cannot remove these files since the process still has a handle on them. I did it this way so that the underlying exception prints in case another exception is raised while removing .ma files. (In short, to avoid an exception mask)